apriltag.c

```diff
@@ -1094,8 +1119,9 @@ zarray_t *apriltag_detector_detect(apriltag_detector_t *td, image_u8_t *im_orig)
                     q->p[j][0] *= td->quad_decimate;
                     q->p[j][1] *= td->quad_decimate;
                 } else {
-                    q->p[j][0] = (q->p[j][0] - 0.5)*td->quad_decimate + 0.5;
-                    q->p[j][1] = (q->p[j][1] - 0.5)*td->quad_decimate + 0.5;
+                    // Maybe merged with the above if? They are doing the same thing now.
```

Comment:
@christian-rauch  Please consider this comment.

Reply:
Do you mean merging the `td->quad_decimate == 1.5` if/else blocks into one?

Reply:
Yes. I'll merge the two blocks in the next commit if you think it's ok too.

Reply:
The if/else blocks are merged in the latest (squashed) commit.